src/views/Admin/AdminEditPermissions.js

```diff
@@ -114,17 +121,11 @@ export default function AdminEditPermissions() {
       </Grid.Item>
       <Grid.Divider />
 
-      {proxyAction(PROXY_TYPE.MANAGEMENT, details.managementProxy, canManage)}
-      <Grid.Divider />
+      {proxyAction(PROXY_TYPE.MANAGEMENT, details.managementProxy, isOwner)}
```

Comment:
are these really only available to owners? a transfer proxy can't change transfer proxy? not even a management key can change proxies?

Reply:
If a management key could change the transfer proxy then we'd have a problem. (^:

Hindsight does tell us that proxies should probably be able to change themselves, but that hasn't been officially proposed yet, let alone implemented/voted on.